src/MeetOne.ts

```diff
@@ -33,16 +33,6 @@ export class MeetOne extends Authenticator {
     super(chains)
   }
 
-  private isMeetOneReady(): boolean {
-    // @ts-ignore
-    if (window.scatter && window.scatter.isInject) {
```

Comment:
Because the `MEET.ONE` injected code which to simulate scatter protocol actions will be delayed.